modules/Microsoft.Network/expressRouteGateway/deploy.bicep

```diff
@@ -0,0 +1,69 @@
+@description('Required. Name of the Express Route Gateway.')
+param name string
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Tags of the Firewall policy resource.')
+param tags object = {}
+
+@description('Optional. Configures this gateway to accept traffic from non Virtual WAN networks.')
+param allowNonVirtualWanTraffic bool = false
+
+@description('Optional. Maximum number of scale units deployed for ExpressRoute gateway.')
+param maxAutoScale int = 2
+
+@description('Optional. Minimum number of scale units deployed for ExpressRoute gateway.')
+param minAutoScale int = 2
+
+@description('Optional. List of ExpressRoute connections to the ExpressRoute gateway.')
+param expressRouteConnections array = []
+
+@description('Required. Resource ID of the Virtual Wan Hub.')
+param virtualHubId string
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (enableDefaultTelemetry) {
+  name: 'pid-47ed15a6-730a-4827-bcb4-0fd963ffbd82-${uniqueString(deployment().name, location)}'
+  properties: {
+    mode: 'Incremental'
+    template: {
+      '$schema': 'https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#'
+      contentVersion: '1.0.0.0'
+      resources: []
+    }
+  }
+}
+
+resource expressRouteGateway 'Microsoft.Network/expressRouteGateways@2022-07-01' = {
+  name: name
+  location: location
+  tags: tags
+  properties: {
+    allowNonVirtualWanTraffic: allowNonVirtualWanTraffic
+    autoScaleConfiguration: {
+      bounds: {
+        max: maxAutoScale
+        min: minAutoScale
```

Comment:
```suggestion
        min: autoScaleConfigurationBoundsMin
```

Reply:
I'd argue this is more aligned with other resources. However, it would require an update of the parameter & readme

Reply:
agreed!